cookbooks/bcpc-hadoop/recipes/hive_hcatalog.rb

```diff
@@ -53,15 +53,6 @@
   user 'hdfs'
 end
 
-bash 'create-hive-scratch' do
-  code <<-EOH
-  hdfs dfs -mkdir -p #{node['bcpc']['hadoop']['hive']['scratch']['dir']}
-  hdfs dfs -chmod 1777 #{node['bcpc']['hadoop']['hive']['scratch']['dir']}
-  hdfs dfs -chown hive:hdfs #{node['bcpc']['hadoop']['hive']['scratch']['dir']}
-  EOH
-  user 'hdfs'
-end
```

Comment:
Pretty sure this block is still required. We want to ensure the directory exists and is owned/permed properly. Obviously we'll have to replace `1777` with something more sane like `0700` and `hive:hdfs` with the proper user/group combo we use for user dirs.